openstates/wi/__init__.py

```diff
@@ -134,6 +134,7 @@
                       'events', 
                       'influenceexplorer'],
     '_ignored_scraped_sessions': [
+        'January 2017 Special Session',
```

Comment:
mind adding a comment to explain why this was skipped, I figure this will raise a red flag at some point since it looks like a relatively current session being skipped (just seeing this I'm curious about Feb. '15 myself)